server/src/main/java/io/spine/server/delivery/memory/InMemoryShardedWorkRegistry.java

```diff
@@ -52,8 +53,18 @@
     private final Map<ShardIndex, ShardSessionRecord> workByNode = newConcurrentMap();
 
     @Override
-    public synchronized Optional<ShardProcessingSession> pickUp(ShardIndex index, NodeId nodeId) {
-        return super.pickUp(index, nodeId);
+    public synchronized Optional<ShardProcessingSession> pickUp(ShardIndex index, NodeId node) {
+        return super.pickUp(index, node);
+    }
+
+    @Override
+    protected WorkerId currentWorkerFor(NodeId node) {
+        WorkerId worker = WorkerId
+                .newBuilder()
+                .setNodeId(node)
+                .setValue(String.valueOf(Thread.currentThread().getId()))
```

Comment:
Please extract the thread ID into a variable. And document the details in the `currentWorkerFor` Javadoc.